### PR TITLE
Update Legislation

### PR DESCRIPTION
**Pull request by StupidCodeFactory**
## Checklist:

- [ ] Automated checks are passing locally.
- [ ] CHANGELOG updated if change is worth telling users about.
### General testing
- [ ] Test without javascript
- [ ] Test on small screen
### Accessibility testing
- [ ] Works keyboard only
- [ ] Tested with one screen reader
- [ ] Zoom page to 400% - content still visible
- [ ] Disable css - does content make sense and appear in a logical order?
- [ ] Passes automated checker (automated in build or manual)

### CODE REVIEW

File: psd-web/config/constants/legislation_constants.yml [hunk 1]
```diff
@@ -1,27 +1,47 @@
...
+  - Explosives Regulations 2014
   - Fireworks Act 2003 / Fireworks Regulations 2004
-  - Noise Emission in the Environment by Equipment for Use Outdoors Regulations 2001
-  - Plugs and Sockets (Safety) Regulations 1994
   - Food Imitations (Safety) Regulations 1989
-  - Furniture and Furnishings (Fire) (Safety) Regulations 1988
-  - Consumer Protection Act 1987 (Part 1)
-  - Nightwear Safety Regulations 1985
-  - Oil Heaters (Safety) Regulations 1977
+  - Furniture and Furnishings (Fire)(Safety) Regulations 1988
```
Comment (LMallows) [inline psd-web/config/constants/legislation_constants.yml]: Furniture and Furnishings (Fire) (Safety) Regulations 1988

Minor change on content to add a space between brackets

File: psd-web/db/migrate/20200121154252_migrate_to_new_legislation.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class MigrateToNewLegislation < ActiveRecord::Migration[5.2]
+  NEW_LEGISTLATION_MAPPING = {
+    "Equipment and Protective Systems Intended for Use in Potentially Explosive Atmospheres Regulations 2016" => "ATEX 2016",
+    "Consumer Protection Act 1987 (Part 1)" => "Consumer Protection Act 1987",
+    "Aerosol Dispensers Regulations 2009" => "Aerosol Dispensers Regulations 2009 (Consumer Protection Act 1987)",
+    "Cosmetic Products Enforcement Regulations 2013 / EC 1223/2009 Cosmetic Products" => "Cosmetic Products Enforcement Regulations 2013",
+    "Furniture and Furnishings (Fire) (Safety) Regulations 1988" =>  "Furniture and Furnishings (Fire)(Safety) Regulations 1988",
```
Comment (LMallows) [inline psd-web/db/migrate/20200121154252_migrate_to_new_legislation.rb]: Furniture and Furnishings (Fire) (Safety) Regulations 1988

Minor change on content to add a space between brackets

Comment (StupidCodeFactory) [reply psd-web/db/migrate/20200121154252_migrate_to_new_legislation.rb]: So essentially this one stay the same

Comment (LMallows) [reply psd-web/db/migrate/20200121154252_migrate_to_new_legislation.rb]: 👍

**Pull request merged**